_posts/2023-09-10-python-multiple-arithmetic-operators.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "The surprising behavior of multiple + and - operators in Python"
```

Comment:
Cut some words out if you can. Attention spans are low out here!

Maybe some clickbait? Not sure what your goals are

Reply:
Is Python the new JavaScript? 😱

Reply:
Is the new one better?
I always struggle with titles.